cogs/commands/karma.py

```diff
@@ -473,13 +485,32 @@ async def plot(self, ctx: Context, *, args: str):
         file = File(img, filename=filename)
         await ctx.send(f"Here you go, {name}! {emoji}", embed=embed, file=file)
 
+    @karma.command(help="Plots the karma change over time of the given karma topic(s)")
+    @commands.cooldown(5, 60, BucketType.user)
+    async def plot(self, ctx: Context, *, args: str):
+        await self.common_plot(ctx, args, False)
+
     @plot.error
```

Comment:
Add xkcd error here too (might have to move after main xkcd)
```suggestion
    @plot.error
    @xkcd.error
```